scipy/stats/_multivariate.py

```diff
@@ -5794,64 +5843,40 @@ class dirichlet_multinomial_gen(multi_rv_generic):
 
     Get the PMF
 
-    >>> import numpy as np
-    >>> alpha = np.array([3, 4, 5])
```

Comment:
No need to make these all arrays. In fact, they need to work with lists, and they didn't before.